bundler/lib/bundler/gem_helper.rb

```diff
@@ -102,19 +106,25 @@ def install_gem(built_gem_path = nil, local = false)
       Bundler.ui.confirm "#{name} (#{version}) installed."
     end
 
-    def build_checksum(built_gem_path = nil)
+    def build_checksums(built_gem_path = nil)
       built_gem_path ||= build_gem
-      SharedHelpers.filesystem_access(File.join(base, "checksums")) {|p| FileUtils.mkdir_p(p) }
-      file_name = "#{File.basename(built_gem_path)}.sha512"
       require "digest/sha2"
-      checksum = ::Digest::SHA512.file(built_gem_path).hexdigest
+      CHECKSUMS.each do |extension, digest_klass|
```

Comment:
Super minor but I'm thinking, do you think the `CHECKSUMS` constant buys us much, over

```ruby
write_checksum(built_gem_path, "sha256", Digest::SHA256)
write_checksum(built_gem_path, "sha512", Digest::SHA512)
```

Reply:
It doesn't get us much, agreed.